README.md

```diff
@@ -1,94 +1,127 @@
 # Discover
 Yet another Discord overlay for Linux written in Python using GTK3.
 
+> **Note:**  
```

Comment:
Changes should not be noted here, README should only show the current correct instructions and information, and anyone wanting to know about changes over time can be expected to read through past commits changing README.